dist/tokens/evo-dark.css

```diff
@@ -9,6 +9,7 @@
         --color-background-confirmation: var(--color-green-4);
         --color-background-information: var(--color-blue-4);
         --color-background-education: #2f373c;
+        --color-background-elevated: var(--color-neutral-7);
```

Comment:
It doesn't look like any of the changes in `evo-dark` or `evo-light` are used anywhere, I know they're probably for a future PR where we overhaul the dropdown but for now I think they should be removed since they are unused.

Reply:
Good catch. I had removed the combobox overrides and forgot to remove these.